.github/workflows/test.yml

```diff
@@ -12,22 +12,63 @@ jobs:
       ROS_DISTRO: ${{ matrix.ros_distro }}
       RMW_IMPLEMENTATION: rmw_cyclonedds_cpp
     container:
-      image: rostooling/setup-ros-docker:ubuntu-jammy-latest
+      image: rostooling/setup-ros-docker:ubuntu-jammy-ros-rolling-ros-base-latest
     strategy:
       fail-fast: false
       matrix:
-          ros_distro: [iron]
+          ros_distro: [rolling]
     steps:
+    - name: apt update
+      run: sudo apt update
+    - name: Install nodejs
+      run: sudo apt install nodejs -y
     - uses: actions/checkout@v2
     - name: Install Cyclone DDS
       run: sudo apt install ros-${{ matrix.ros_distro }}-rmw-cyclonedds-cpp -y
+    - name: Install geometry_msgs
```

Comment:
This should _not_ be necessary to call out each individual dependency, this is what rosdep is for

Reply:
> CI is still failing, unfortunately, very early in this process with basic dependency resolution - please look into that!
I understand that rosdep is the way these items *SHOULD* be installed but rosdep was/is still?? not working for rolling on 22.04/jammy which is why I went in this direction . I know it is ugly and wrong and could be done in one step but I was taking bites at it one error at a time, the action build time getting progressively longer, now at 1h12m.   This is how I actually made progress in my own environments when rosdep got broken.  I use ansible to install the packages required.

There are currently no packages for ros-rolling-nav2* so this opennav_coverage with behavior tree v4 needs to build it.    Iron uses behaviourtree_cpp_v3 https://github.com/ros-navigation/navigation2/blob/4e16d3c7e3cfac08c4b79841f19570c40ebbe14b/nav2_behavior_tree/package.xml#L20 so not an option.

The rosdep workaround ["Attempt to fix Rolling 22.04 CI Until 24.04 Comes of Age"](https://github.com/ros-navigation/navigation2/commit/80bb5bff1488c0677efcc4254b7a89908c853ba0) that ros-planning/navigation2 uses was not immediately obvious on how or if it even should be applied to the opennav_coverage repository.  I assume duplicating that infra is not what you want, especially as its likely that the contents of this repository will be moved into nav2, soon after fields2cover packages correctly.

Is there is a better/faster solution where opennav_coverage can use a ci docker image of a built/installed rolling nav2 as a starting point?

In attempting to debug all this I was not able to get a local github actions environment setup. Do you use that? 
```
https://nektosact.com/introduction.html
It doesn't work locally for me. If fails on
vcs import --force --recursive src/ < package.repo
Could not checkout ref 'a077eb600a3700d1053482b3582822fa1b6e48db':
fatal: reference is not a tree: a077eb600a3700d1053482b3582822fa1b6e48db
```

NOTE: Before this is merged I also need to revert pointing this at my fork of fields2cover.  I am STILL trying to help fight the build farm to get packages of fields2cover to build  but we all live in different time zones and have other things going on so there is a lag between fix/apply/potentially revert that messes up this PR sometimes. See https://github.com/Fields2Cover/Fields2Cover/issues/136

I will look into how to tweak the action to only run the opennav_coverage tests and not the nav2 and fields2cover tests.

Reply:
Related - Action ignores failed rosdep install https://github.com/ros-tooling/action-ros-ci/issues/864

Reply:
Related - Rolling CI fails  https://github.com/ros-tooling/action-ros-ci/issues/844

Reply:
Am I correct, albeit slow to realise it if so, rolling is no longer going to be supported/function on ubuntu 22.04?  Or is it just in a state of flux that has not yet been worked out?

Reply:
Did you try for Rolling on 24.04? That is what Rolling targets now, so it doesn't surprise me if you run into issues if you're not using a statically set rosdistro version pre-rolling update to 24.04. Instead of Jammy, use the Noble ones I think should fix that and not require specifically calling out every dependency this way. I think that's what the threads point to as well that you linked above! :smile: 

I haven't used the local Actions yet, no :(